index.bs

```diff
@@ -269,6 +269,7 @@ To <dfn id="waiting-for-all" export lt="wait for all|waiting for all">wait for a
 1. Let |index| be 0.
 1. Let |total| be |promises|'s [=list/size=].
 1. Let |result| be a <a>list</a> containing |total| null values.
+1. If |total| is 0, then perform |successSteps| given |result| and abort these steps.
```

Comment:
```suggestion
1. If |total| is 0, then perform |successSteps| given |result| and return.
```

Since this is not a series of substeps, we can just use "return".